### PR TITLE
Kan 51 flag raw data with values below a minimum 5.0 just to remove sdp issues

### PR DESCRIPTION
**Pull request by wkhu-astro**

### CODE REVIEW

File: museek/plugin/rawdata_flagger_plugin.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                             Requirement(location=ResultEnum.OUTPUT_PATH, variable='output_path')]
+
+    def run(self, data: TimeOrderedData, output_path: str):
+        """
+        Flag raw data with values below a minimum
+        :param data: time ordered data of the entire block
+        :param output_path: path to store results
+        """
+        data.load_visibility_flags_weights()
+        new_flag = np.zeros(data.shape, dtype=bool)
```
Comment (amadeuswi) [inline museek/plugin/rawdata_flagger_plugin.py:41]: it's safer to do `data.visibility.shape`
In this case it's the same, but `data.shape` is always the complete data shape and `data.visibility.shape` is only the part that's loaded, e.g. only the scan part or only the calibrator observation.

File: museek/plugin/rawdata_flagger_plugin.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+class RawdataFlaggerPlugin(AbstractPlugin):
+    """ Plugin to flag raw data with values below a minimum """
+
+    def __init__(self,
+                 flag_minimum: float):
+        """
+        Initialise
+        :param flag_minimum: lower threshold to flag the data
```
Comment (amadeuswi) [inline museek/plugin/rawdata_flagger_plugin.py:23]: I would prefer a more descriptive parameter name like `flag_lower_threshold`

File: museek/plugin/rawdata_flagger_plugin.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+from ivory.utils.requirement import Requirement
+from ivory.utils.result import Result
+from museek.enums.result_enum import ResultEnum
+from museek.factory.data_element_factory import FlagElementFactory
+from museek.flag_list import FlagList
+from museek.time_ordered_data import TimeOrderedData
+from museek.visualiser import waterfall
+
+
+class RawdataFlaggerPlugin(AbstractPlugin):
```
Comment (amadeuswi) [inline museek/plugin/rawdata_flagger_plugin.py:16]: Please add this plugin in the readme under "available plugins"

File: museek/plugin/rawdata_flagger_plugin.py (hunk 2 above)
Comment (amadeuswi) [inline museek/plugin/rawdata_flagger_plugin.py:23]: please mention in the docstring that this threshold relates to raw correlator units without any normalisation.

**Review by amadeuswi — APPROVED** [review]
All looks good to me, only minor comments.
I assume the plugin runs fine and all, I did not test it.

File: museek/plugin/rawdata_flagger_plugin.py (hunk 2 above)
Comment (wkhu-astro) [reply museek/plugin/rawdata_flagger_plugin.py:23]: changed from flag_minimum to flag_lower_threshold

Comment (wkhu-astro) [reply museek/plugin/rawdata_flagger_plugin.py:23]: mentioned

**Pull request merged**